### ISSUE TITLE
Fix Voice Over on Inactive Tabs section in Tab tray

### ISSUE THREAD

**Issue by lmarceau** [mozilla-mobile/firefox-ios#10326]
# Summary
When Inactive tabs section appears in the tab tray, Voice Over users should have a short explanation of what pressing this section does. Currently "Inactive tabs" text is read but also the image name "menu disclosure" which is not helping the user.

![width=200,height=183](https://user-images.githubusercontent.com/11338480/160490588-93b414d0-1e8c-4473-9306-699be2002167.png)

### When the section is closed
"Tap to open inactive tabs" with the button accessibility trait

### When the section is opened
"Tap to close inactive tabs" with the button accessibility trait

**Also fix:**

* Center label vertically in unopened section
* Make whole section header tappable
* Center icon + label horizontally for Close all button

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-4001)

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Verifying as fix on main 6aaca8a5c4ba01029db4239fffe6fee63465c111. 

Note: 

  * When the section is closed → "View inactive tabs" with the button accessibility trait

  * When the section is opened → "Hide inactive tabs" with the button accessibility trait
